istio-cni/Chart.yaml

```diff
@@ -1,12 +1,12 @@
 apiVersion: v1
 name: istio-cni
-version: 0.2.0
-appVersion: 0.2.0
-tillerVersion: ">=2.7.2-0"
+version: 1.1.0
```

Comment:
Those 3 lines should probably be dropped.

Reply:
done